<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CadastroUsers</class>
 <widget class="QDialog" name="CadastroUsers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>282</width>
     <height>211</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Nome</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>NP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Usuário</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Senha</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="celular_telefone"/>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Área</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLineEdit" name="line_nome"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line_np"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line_user"/>
        </item>
        <item>
         <widget class="QLineEdit" name="line_senha">
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="line_telefone"/>
        </item>
        <item>
         <widget class="QComboBox" name="area"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QCheckBox" name="brigadista">
      <property name="text">
       <string>Brigadista</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="cadastrarButton">
        <property name="text">
         <string>Cadastrar usuário</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="atualizarButton">
        <property name="text">
         <string>Atualizar cadastro</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="excluirButton">
        <property name="text">
         <string>Excluir usuário</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
